config/jobs/kubernetes/kops/build_jobs.py

```diff
@@ -1363,7 +1363,7 @@ def generate_misc():
                        "--set=spec.kubeAPIServer.runtimeConfig=api/all=true",
                        "--gce-service-account=default",
                    ],
-                   kubernetes_feature_gates="AllAlpha,DisableCloudProviders,DisableKubeletCloudCredentialProviders,-EventedPLEG",
+                   kubernetes_feature_gates="AllAlpha,-EventedPLEG",
```

Comment:
/hold

@hakman @justinsb should approve the kops changes

I do not know if this needs to regenerate some files

Reply:
BTW It's `make -C config/jobs/kubernetes/kops`, or `make -C config/jobs/kubernetes/kops generate-jobs` if you don't have AWS credentials.  (Maybe we should make that the default ... though I would quite like a job that would periodically run `invalidate-pins` in that case)

https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/kops/Makefile